lib/archethic/account.ex

```diff
@@ -27,8 +27,9 @@ defmodule Archethic.Account do
       %UnspentOutput{type: :UCO, amount: amount}, acc ->
         Map.update!(acc, :uco, &(&1 + amount))
 
-      %UnspentOutput{type: {:NFT, nft_address}, amount: amount}, acc ->
-        update_in(acc, [:nft, Access.key(nft_address, 0)], &(&1 + amount))
+      %UnspentOutput{from: from, type: {:NFT, nft_address, nft_id}, amount: _amount}, acc ->
+        put_in(acc, [:nft, nft_address], {from, nft_id})
```

Comment:
You still need to keep the amount of the NFT.

Reply:
right